tests/python_generator_expressions/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,101 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`generator_expressions.py 1`] = `
+(x for x in range(100))
+(x for x in range(100) if x % 2 == 0)
+
+my_long_variable_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa = (thing for thing in things)
+
+a = (my_long_function_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa(thing) for thing in things)
+
+a = (my_long_function_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa(thing) for thing in things if thing is not None)
+
+a = (my_long_function_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa(thing) for thing in things if my_long_predicate_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa(thing))
+
+list(x for x in bar)
+
+foo((x for x in bar), True)
+
+foo(True, (x for x in bar))
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+(x for x in range(100))
+
+(x for x in range(100) if x % 2 == 0)
+
+my_long_variable_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa = (
+    thing for thing in things
+)
+
+a = (
+    my_long_function_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa(thing)
+    for thing in things
+)
+
+a = (
+    my_long_function_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa(thing)
+    for thing in things if thing is not None
+)
+
+a = (
+    my_long_function_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa(thing)
+    for thing in things
+    if my_long_predicate_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa(thing)
```

Comment:
Indent this line?

Reply:
PEP8 (or PyCharm's PEP8 checker) seems to complain when I indent that line ([`continuation line unaligned for hanging indent`](https://lintlyci.github.io/Flake8Rules/rules/E131.html)), and I'd defer to PEP8 as the source of truth. But it's always possible PyCharm's PEP8 checker is wrong :) I can look into this a bit more if you'd like!

Reply:
Here’s [the PEP 8 section on indentation](https://www.python.org/dev/peps/pep-0008/#indentation), but I’m not sure how it applies here.

Reply:
I guess the way I interpret that is "things should be aligned with the first character of real content", eg if the first argument or list item or part of a generator expression is on the same line as the open paren, then if it breaks to a new line, that line should start at the same column as that first character on the previous line. If that first character is on a new line (not the same line as the open paren) then it should be indented once, compared to the previous line; but then the same rules apply, that all subsequent lines of that logical group should start at the same column as that first character.

Sorry, that's kinda rambling haha.

For what its worth, `flake8` also doesn't like the additional indent. It's okay with this:
```python
a = (
    i
    for i in range(1)
    if i % 2 == 0
)
```
but complains about this:
```python
a = (
    i
    for i in range(1)
        if i % 2 == 0
)
```

Given that Flake8 is a pretty common tool for conforming to PEP8, we might want to go along with the results that tool gives. Alternatively we could add an option to allow indenting more here :)

What do you think?

Reply:
Going with Flake8 sounds good 👍